das/simple_das_reader_aggregator.go

```diff
@@ -262,6 +265,37 @@ func (a *SimpleDASReaderAggregator) tryGetByHash(ctx context.Context, hash []byt
 }
 
 func (a *SimpleDASReaderAggregator) Start(ctx context.Context) {
+	onlineUrlsChan := StartRestfulServerListFetchDaemon(ctx, a.config.OnlineUrlList, a.config.OnlineUrlListFetchInterval)
```

Comment:
We want to pass to `StartRestfulServerListFetchDaemon` a context that will be automatically cancelled when all of the activity launched by `Start` ends.

Reply:
Done.